b1ddi/data_source_ipam_next_available_subnet.go

```diff
@@ -14,6 +14,17 @@ import (
 )
 
 func dataSourceIpamsvcNextAvailableSubnet() *schema.Resource {
+	subSchema := dataSourceSchemaFromResource(resourceIpamsvcSubnet)
```

Comment:
is `subSchema`used? I think you copied code to a function but forgot to delete here :)

Reply:
Yeah, I missed that ;). Will remove it

Reply:
removed the dead code